.eslintrc

```diff
@@ -1,3 +1,4 @@
 {
+  "parser": "babel-eslint",
```

Comment:
Without this it wasn't using the babel-eslint parser. This fixes the same issue but better.

Reply:
Oh wow yes we should fix this in create-single-spa - I just created https://github.com/single-spa/create-single-spa/issues/37 to track this